static/bidder-info/xeworks.yaml

```diff
@@ -0,0 +1,19 @@
+endpoint: "http://prebid-{{.Host}}.xe.works/?pid={{.SourceId}}"
```

Comment:
We discourage to use endpoints like this. Client specific subdomains prevent Prebid Server from reusing connections across your clients which results in more connections needed for your adapter. The issue gets worse the more successful you are in attracting new clients. Hosts may choose to disable your adapter due to this behavior.
I encourage you to please use params as - 

endpoint: "http://prebid.xe.works/?pid={{.SourceId}}&host={{.Host}}"

Reply:
Hi Ashish. It would be really useful for me to have a macro in the hostname. I'm going to use it to run tests and won't be balancing requests between clients using that mechanism. Therefore, the total number of hosts to xeworks on the running instance will always be 1. If it's required to remove this macro, please let me know.

Reply:
I would still recommend to move the host name to query param instead of using it in the uri.